api/server/server_unix.go

```diff
@@ -38,6 +39,9 @@ func (s *Server) newServer(proto, addr string) ([]*HTTPServer, error) {
 	case "unix":
 		l, err := sockets.NewUnixSocket(addr, s.cfg.SocketGroup)
 		if err != nil {
+			if strings.Contains(err.Error(), "is a directory") {
```

Comment:
I think we can simplify this.
Instead of looking into the error like this, we can do:
`return nil, fmt.Errorf("can't create unix socket %s: %v, addr, err)`